pycaret/internal/preprocess/transformers.py

```diff
@@ -585,7 +585,11 @@ def transform(self, X, y):
             y_new.index = new_index
 
             # Add the new samples to the old dataframe
-            X, y = X.append(X_new), y.append(y_new)
+            # X, y = X.append(X_new), y.append(y_new)
```

Comment:
don't comment it out. just replace the old line with the two new ones. Also, the comment isn't necessary. You don't need to comment on changes in syntax.